mythril/laser/ethereum/instructions.py

```diff
@@ -1894,17 +1971,16 @@ def callcode_post(self, global_state: GlobalState) -> List[GlobalState]:
         return [global_state]
 
     @StateTransition()
-    def delegatecall_(self, global_state: GlobalState) -> List[GlobalState]:
+    def staticcall_(self, global_state: GlobalState) -> List[GlobalState]:
         """
 
         :param global_state:
         :return:
         """
         instr = global_state.get_current_instruction()
         environment = global_state.environment
-
         try:
-            callee_address, callee_account, call_data, value, gas, _, _ = get_call_parameters(
+            callee_address, callee_account, call_data, value, gas, memory_out_offset, memory_out_size = get_call_parameters(
```

Comment:
The EIP https://github.com/ethereum/EIPs/pull/214/files#diff-29d8cb8d19a769c141160ce5c1e65caeR31 claims that there are only 6 parameters for STATICCALL and value is always 0.

Reply:
In this case it returns 0. there is the parameter `with_value` which is False in this case.